resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -749,5 +761,13 @@ public CircuitBreakerConfig build() {
                 .orElseGet(() -> recordExceptionPredicate != null ? recordExceptionPredicate
                     : DEFAULT_RECORD_EXCEPTION_PREDICATE);
         }
+
+        private IntervalFunction validateWaitIntervalFunctionInOpenState() {
+            if (setWaitInOpenStateCounter > 1) {
+                throw new IllegalStateException("You have already the durationInOpenState value from the waitDurationInOpenState method." +
```

Comment:
Could you please change the message to:
"The waitIntervalFunction was configured twice which could result in an undesired state. Please use either waitIntervalFunctionInOpenState or  waitDurationInOpenState."